tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/EagerOperationBuilder.java

```diff
@@ -55,6 +55,9 @@
 final class EagerOperationBuilder implements OperationBuilder {
 
   EagerOperationBuilder(EagerSession session, String type, String name) {
+    if(!session.isOpEnabled(type))
+      throw new IllegalArgumentException("Op " + type + " is not valid in eager mode.");
```

Comment:
can you please add a space after `if` and also wrap the `throw` statement in a block `{...}`? This is more in line with the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used) that we follow in this project.